src/sentry/static/sentry/app/utils/dates.tsx

```diff
@@ -7,7 +7,7 @@ export const DEFAULT_DAY_START_TIME = '00:00:00';
 export const DEFAULT_DAY_END_TIME = '23:59:59';
 const DATE_FORMAT_NO_TIMEZONE = 'YYYY/MM/DD HH:mm:ss';
 
-function getParser(local = false) {
+function getParser(local: boolean = false): Function {
```

Comment:
I think we can refine the return type: 

```suggestion
function getParser(local: boolean = false): typeof moment | typeof moment.utc {
```

The return type can be gnarly, but it should ensure the right function signature is propagated from `getParser()`.

Reply:
Nice catch 👍

Reply:
Wait... does this work... I didn't know you can use typeof like this

Reply:
Yep you can do it like that. It's super useful.

In flow, `typeof` works in a similar manner.